File: Month 1/intro to java/day 1/src/Java1/Ques3.java
package Java1;

import java.util.Scanner;

public class Ques3 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String s1;
        char s2;
        System.out.println("Enter the string: ");
        s1 = sc.nextLine();
        System.out.println("Enter the character whose occurence to be calculated: ");
        s2 = sc.next().charAt(0);

        int charcount = s1.length() - s1.replaceAll(""+ s2, "").length();
        System.out.println("Occurrence Of the Char In String: " + charcount);

    }
}

File: Bootcamp/Android/intro to kotlin 1/src/Ques 3.kt
import java.util.*

/*

Write a program to find the number of occurrences of a character in a string without using loop.


*/

fun main() {

    val scanner = Scanner(System.`in`)
    var string = ""
    var character:Char

    println("Enter the String :  ")
    string = scanner.nextLine()

    println("Enter the Character")

    character = scanner.next().toCharArray().get(0)

    val charCount = string.length - string.replace(""+character,"").length
    println("Occurance of Character is : $charCount")


}